js/src/ui/LanguageSelector/languageSelector.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component } from 'react';
+import { MenuItem } from 'material-ui';
+
+import Select from '../Form/Select';
+import Translate from '../Translate';
+import { getLocale, setLocale } from '../../i18n';
+
+const LOCALES = [
+  'en', 'de'
```

Comment:
There is a need for both - when WIP, you want to see stuff (so probably have a list of all), but not necessarily have it live. 

How that works out, don't know yet, and not overthinking it either. Will get there, may be as simple as dev vs prod where the one shows all, the other one not